docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
I think this was mentioned in Monday's standup, but how exactly are images updated when there is a new release? And how do we know that an image tag reflects a certain release? I see that 'pip install volttron' is done in setup-platform.py and recall reading the email thread about that decision (which I still don't understand).

Reply:
This was what @shwethanidd and I decided. 

If volttron is installed in the base container then EVERYTHING needs to be installed in the base container because when you restart a container you lose the whl state.

You still have the home directory, but you don't have any of the code that is installed in the site-packages.  Using pip.conf file we could put the location for site-packages wherever we want, however when we install an agent it will download another copy of volttron to be in the new site-packages.  So now we have 2 copies of volttron.

So we made the decision to just have a wrapper container.  Actually I am not 100% sure we need to update the base volttron container after it has been built.